packages/eslint-config-nimble-testing/package.json

```diff
@@ -13,14 +13,13 @@
   "author": "Nimble",
   "main": "lib/index.js",
   "dependencies": {
-    "@nimblehq/eslint-config-nimble-core": "^2.4.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-prettier": "^4.2.1"
-  },
-  "peerDependencies": {
-    "eslint": "^8.12.0",
-    "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-jest": "^27.2.1"
+    "@nimblehq/eslint-config-nimble-core": "^2.7.0",
+    "eslint": "^8.51.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-cypress": "^2.15.1",
```

Comment:
I think we can remove this package, and we can create another one for the e2e test later. Because some projects didn't use UI tests e.g. infrastructure-templates

Reply:
I removed it in cc4197f ✅